.github/workflows/main.yml

```
@@ -0,0 +1,26 @@
+name: Ansible Lint
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    # Important: This sets up your GITHUB_WORKSPACE environment variable
+    - uses: actions/checkout@v2
+
+    - name: Lint Ansible Playbook
+      # replace "master" with any valid ref
+      uses: ansible/ansible-lint-action@master
+      with:
+        # [required]
+        # Paths to ansible files (i.e., playbooks, tasks, handlers etc..)
+        # or valid Ansible directories according to the Ansible role
+        # directory structure.
+        # If you want to lint multiple ansible files, use the following syntax
+        # targets: |
+        #   playbook_1.yml
+        #   playbook_2.yml
+        targets: "wordpress-nginx/site.yml"
```

From https://github.com/ansible/ansible-examples/blob/6e7dd3d3613e1448703a2137ff892c2da6196e97/.github/main.workflow#L9

---

I think the linter offenses should be fixed before merging this

---

Also, YAML doesn't need quotes
```suggestion
        targets: wordpress-nginx/site.yml
```

---

> the linter offenses should be fixed before merging this
Opened https://github.com/ansible/ansible-examples/issues/271

---

@webknjaz Opened https://github.com/ansible/ansible-examples/pull/278 to fix linter offenses!